app/react/Forms/components/MultiSelect.tsx

```diff
@@ -317,24 +317,27 @@ abstract class MultiSelectBase<ValueType> extends Component<
   renderOption(option: Option, index: number, groupIndex = '') {
     const { optionsValue, optionsLabel, prefix } = this.props;
     const key = `${groupIndex}${index}`;
+    const isNotAnyOption = option.id !== 'any';
     return (
-      <li
-        className="multiselectItem"
-        key={key}
-        title={option.title ? option.title : option[optionsLabel]}
-      >
-        <input
-          type="checkbox"
-          className={`multiselectItem-input${
-            this.checked(option) === SelectStates.PARTIAL ? ' partial' : ''
-          }`}
-          value={option[optionsValue]}
-          id={prefix + option[optionsValue]}
-          onChange={this.change.bind(this, option)}
-          checked={this.checked(option) !== SelectStates.OFF}
-        />
-        {this.label(option)}
-      </li>
+      isNotAnyOption && (
```

Comment:
@Zasa-san why did you decide to exclude the 'any' option here? Given the issue is related only to the filters, isn't the  [FiltersFromProperties](https://github.com/huridocs/uwazi/blob/109cd29aae9732f05ade623a7dd0e068784754cd/app/react/Library/components/FiltersFromProperties.js#L35) component a better place to exclude the option?

Reply:
It makes sense to exclude it in the FiltersFromProperties component.